coderd/database/dbmem/dbmem.go

```diff
@@ -905,6 +905,15 @@ func (*FakeQuerier) AcquireLock(_ context.Context, _ int64) error {
 	return xerrors.New("AcquireLock must only be called within a transaction")
 }
 
+func (*FakeQuerier) AcquireNotificationMessages(_ context.Context, arg database.AcquireNotificationMessagesParams) ([]database.AcquireNotificationMessagesRow, error) {
+	err := validateDatabaseType(arg)
+	if err != nil {
+		return nil, err
+	}
+	// nolint:nilnil // Irrelevant.
```

Comment:
> 	// nolint:nilnil // Irrelevant.

Is it present somewhere later in the stack?

Reply:
Yup, in the next PR all of these are implemented; I just stubbed these out so CI passes.